pkg/pod/status_test.go

```diff
@@ -233,6 +391,7 @@ func TestMakeTaskRunStatus(t *testing.T) {
 						}},
 					Name:      "state-name",
 					Container: "step-state-name",
+					Results:   []v1.TaskRunResult{},
```

Comment:
I think we don't need add it for every case, there is a `// Common traits, set for test case brevity.` below where you can set it there?

Reply:
It's not the same everywhere. For some test cases, it is actually populated.

Reply:
I fixed it though :)